src/index.ts

```diff
@@ -156,24 +161,34 @@ export class Spring {
     // being changed in `updatedConfig`, we run the simulation with `_step()`
     // and default `fromValue` and `initialVelocity` to their current values.
 
-    this._advanceSpringToTime(Date.now());
+    this._advanceSpringToTime();
 
-    const baseConfig = {
-      fromValue: this._currentValue,
-      initialVelocity: this._currentVelocity
-    };
+    this._config.fromValue = this._currentValue;
+    this._config.initialVelocity = this._currentVelocity;
 
-    this._config = {
-      ...this._config,
-      ...baseConfig,
-      ...updatedConfig
-    };
```

Comment:
Not sure why you felt the need to change this.  Was there a problem, or do you just prefer the iterative style?

Reply:
https://github.com/skevy/wobble/issues/9